auth0/src/main/java/com/auth0/android/authentication/storage/SecureCredentialsManager.java

```diff
@@ -310,4 +361,42 @@ long getCurrentTimeInMillis() {
         return clock.getCurrentTimeMillis();
     }
 
+    private boolean hasScopeChanged(@NonNull String storedScope, @Nullable String requiredScope) {
```

Comment:
Why not put these methods in an abstract class that both CredentialsManager can inherit from? That will remove this code duplication.

Reply:
Done in a new separate commit.